code/game/machinery/spaceheater.dm

```diff
@@ -3,8 +3,8 @@
 	density = 1
 	icon = 'icons/obj/atmos.dmi'
 	icon_state = "sheater0"
-	name = "space heater"
-	desc = "Made by Space Amish using traditional space techniques, this heater is guaranteed not to set the station on fire."
+	name = "space A/C"
```

Comment:
Make sure you've tested to see if the slash character breaks anything. It shouldn't, but who knows, with BYOND.

Reply:
I did and it worked fine :)

Reply:
Just did a test and it works fine